spalp.module

```diff
@@ -37,6 +39,22 @@ function spalp_theme() {
   ];
 }
 
+/**
+ * Implements hook_entity_view().
+ */
+function spalp_entity_view(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display, $view_mode) {
+
+  // Are we viewing an applanding node?
+  if ($entity->bundle() == 'applanding' && $view_mode == 'full') {
+    // Get the app ID.
+    $app_id = $entity->get('field_spalp_app_id')->getString();
+
+    // TODO: Check if the app's module defines a library?
```

Comment:
from a quick Google, I haven't found a way of doing this - any suggestions?

Reply:
possibly \Drupal\Core\Asset\LibraryDiscoveryInterface::getLibrariesByExtension

Reply:
There might be some inspiration here in this code. This is how core test tries to get all the libs for modules:
```
  protected function getAllLibraries() {
    $modules = \Drupal::moduleHandler()->getModuleList();
    $extensions = $modules;
    $module_list = array_keys($modules);
    sort($module_list);
    $this->assertEqual($this->allModules, $module_list, 'All core modules are installed.');

    $themes = $this->themeHandler->listInfo();
    $extensions += $themes;
    $theme_list = array_keys($themes);
    sort($theme_list);
    $this->assertEqual($this->allThemes, $theme_list, 'All core themes are installed.');

    $libraries['core'] = $this->libraryDiscovery->getLibrariesByExtension('core');

    $root = \Drupal::root();
    foreach ($extensions as $extension_name => $extension) {
      $library_file = $extension->getPath() . '/' . $extension_name . '.libraries.yml';
      if (is_file($root . '/' . $library_file)) {
        $libraries[$extension_name] = $this->libraryDiscovery->getLibrariesByExtension($extension_name);
      }
    }
    return $libraries;
  }
```

Reply:
thanks - for alpha, I think we can assume that all extending modules will provide a library

I've created https://www.drupal.org/project/spalp/issues/2996137 for adding this check